ro-base.obo

```diff
@@ -800,78 +807,78 @@ def: "w 'has process component' p if p and w are processes,  w 'has part' p and
 domain: BFO:0000015 ! process
 range: BFO:0000015 ! process
 is_a: RO:0002180 ! has component
-created_by: dos
+created_by: https://orcid.org/0000-0002-7073-9172
 creation_date: 2017-05-24T09:49:21Z
 
 [Typedef]
 id: RO:0002019
 name: has ligand
 def: "A relationship that holds between between a receptor and an chemical entity, typically a small molecule or peptide, that carries information between cells or compartments of a cell and which binds the receptor and regulates its effector function." []
 is_a: RO:0002233 ! has input
-created_by: dos
+created_by: https://orcid.org/0000-0002-7073-9172
 creation_date: 2017-07-19T17:30:36Z
 
 [Typedef]
 id: RO:0002020
 name: transports
 def: "Holds between p and c when p is a transport process or transporter activity and the outcome of this p is to move c from one location to another." []
 is_a: RO:0002313 ! transports or maintains localization of
-created_by: dos
+created_by: https://orcid.org/0000-0002-7073-9172
 creation_date: 2017-07-20T17:11:08Z
 
 [Typedef]
 id: RO:0002021
 name: occurs across
 def: "A relationship between a process and a barrier, where the process occurs in a region spanning the barrier. For cellular processes the barrier is typically a membrane. Examples include transport across a membrane and membrane depolarization." []
 is_a: RO:0002479 ! has part that occurs in
-created_by: dos
+created_by: https://orcid.org/0000-0002-7073-9172
 creation_date: 2017-07-20T17:19:37Z
 
 [Typedef]
 id: RO:0002022
 name: directly regulated by
-comment: Process(P2) is directly regulated by process(P1) iff: P1 regulates P2 via direct physical interaction between an agent executing P1 (or some part of P1) and an agent executing P2 (or some part of P2).  For example, if protein A has protein binding activity(P1) that targets protein B and this binding  regulates the kinase activity (P2) of protein B then P1 directly regulates P2. {xref="GOC:dos"}
+comment: Process(P2) is directly regulated by process(P1) iff: P1 regulates P2 via direct physical interaction between an agent executing P1 (or some part of P1) and an agent executing P2 (or some part of P2).  For example, if protein A has protein binding activity(P1) that targets protein B and this binding  regulates the kinase activity (P2) of protein B then P1 directly regulates P2. {xref="https://orcid.org/0000-0002-7073-9172"}
```

Comment:
Is it correct that in a comment we write `{xref="https://orcid.org/0000-0002-7073-9172"}` and the same thing looks like `[https://orcid.org/0000-0002-7073-9172]` in a `definintion`? (see below)

Reply:
yes, this is how OBO works.